events.md

```diff
@@ -4,13 +4,13 @@ sidebar_link: true
 sidebar_sort_order: 100
 ---
 
-Boston Python runs four kinds of events:
+Boston Python runs several kinds of events, most of which are listed on the [Boston Python meetup page](http://bostonpython.com). RSVP there to be sure to get a spot.
 
+- [**Office hour**](officehour.md) is a weekly hour to drop in and ask anything on your mind.
 - **Presentation nights** feature presentations by members.  These are technical peer-to-peer presentations.  Anyone can do one!  Sometimes we do [lightning talks](lightning.md).
-- **Collaboration nights** are unstructured times to sit with people with similar interests, and do whatever you would like to do.
 - **Python Study Group nights** feature presentations by members who are beginner-to-intermediate. Each presenter chooses a topic that they're unfamiliar with, researches it, and then teaches the group what they've learned.
-- [**Office hour**](officehour.md) is a weekly hour to drop in and ask anything on your mind.
-
-Specific events are listed on the [Boston Python meetup page](http://bostonpython.com).  RSVP there to be sure to get a spot.
+- **Book Discussion nights** host discussion focused on particular books. As of March 2023, this group is currently discussing **Fluent Python (2nd ed.)** on Monday evenings. (Note: this group does **not** post events on Meetup, join us on Slack to participate!)
```

Comment:
This content will be stale at some time in the near future. Should we set an alarm to update it, or should we rewrite it without the time-dependent content so we can leave it alone?

For example, we could say something like `**Book Discussions** are Boston Python's book club meetings. Typically, we take our time to dig into the content, so we'll often devote a session to one chapter of a book. See the #book-discussion channel on Slack if you'd like to know more."

Perhaps we should ask Glenn to weigh in on the wording.